vendor/github.com/mendersoftware/mender-artifact/areader/reader.go

```diff
@@ -34,6 +34,7 @@ import (
 )
 
 type SignatureVerifyFn func(message, sig []byte) error
+type ShouldBeSignedFn func(artifact.HeaderInfoer) bool
```

Comment:
For simplicity and review purposes we can keep these changes here now, but before this is merged, all vendor files needs to be submitted as a separate pull request in https://github.com/mendersoftware/mender-artifact, and then they need to be update using the go mod vendor mechanism.

Reply:
Sounds good. I'm not familiar with the standard go tools (I usually use bazel in a monorepo), could I get some pointers? I see `# github.com/mendersoftware/mender-artifact v0.0.0-20220913084855-9ed8ad0d53d0` in modules.txt - how do I generate a new version for this when we're ready?

Reply:
I ended up removing all my areader changes, so I no longer need this. But, I do have another PR that is in a similar situation (https://github.com/Michaelhobo/mender-artifact-1/pull/2 ), so I'd like some pointers anyway.